scripts/karma.conf.js

```diff
@@ -0,0 +1,20 @@
+const generate = require('videojs-generate-karma-config');
+
+module.exports = function(config) {
+  config = generate(config);
+
+  // any custom stuff here!
+
+  // We only test on ChromeHeadless
+  // which means the travis config in this repo is also
+  // different
+  config.plugins.push('karma-coverage');
+  config.browsers = ['ChromeHeadless'];
```

Comment:
We can use the vjs karma defaults here to be more consistent (and update the travis file). This was only running in headless Chrome because it's a relatively vanilla library.

Reply:
OK I will revert this and the travis file too